docs/user/use_pretrained_network_to_start_new_experiment.md

```diff
@@ -0,0 +1,80 @@
+# Use a pre-trained network to start a new experiment
+
+In this section you will learn how to load a pre-trained network and use it in
+new experiments. We'll cover two cases in particular:
```

Comment:
Can you add a general explanation here, and then use the two cases below as examples?

Reply:
Done. Is what I added sufficient?